a.core_concepts.md

```diff
@@ -290,7 +290,7 @@ kubectl describe nginx | grep val1
 
 # or
 
-kubectl run po busybox --restart=Never --image=busybox --env=var1=val1 -it --rm -- env
+kubectl run nginx --restart=Never --image=busybox --env=var1=val1 -it --rm -- env
```

Comment:
this should be `kubectl run busybox`, since the image is 'busybox', no?

Reply:
Actually in the question says `nginx`, so I should fix the image..
![image](https://user-images.githubusercontent.com/1859450/50822047-033a7a00-1331-11e9-9b34-8403b208c115.png)
Thanks for spotting 👁

Reply:
oh, yes, you're absolutely right! thanks!